try:
    print(unknown_variable)
except NameError as e:
    print("Caught an error:", e)
